components/TypeForm/PopUp.tsx

```diff
@@ -1,22 +1,21 @@
 'use client';
 import { PopupButton } from '@typeform/embed-react';
 import { type PopupOptions } from '@typeform/embed/types';
-import { cnb, type ClassValue } from 'cnbuilder';
 
 export type PopUpProps = PopupOptions & {
   id: string;
   children: React.ReactNode;
-  className?: ClassValue;
+  className?: string;
 }
 
 const PopUp = ({
   id, children, className, ...rest
 }:PopUpProps) => {
   return (
-    <PopupButton noHeading keepSession id={id} className={cnb(className)} {...rest}>
+    <PopupButton buttonProps={{ type: 'button' }} noHeading keepSession id={id} className={className} {...rest}>
```

Comment:
Pass in `type="button"` to the TypeForm component pop up CTA since it doesn't have that property out of the box.